app/src/main/java/tech/ula/ui/FilesystemListFragment.kt

```diff
@@ -50,7 +66,7 @@ class FilesystemListFragment : Fragment() {
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-        filesystemListViewModel.getAllFilesystems().observe(viewLifecycleOwner, filesystemChangeObserver)
+        filesystemListViewModel.getAllFilesystems().observe(this  ,filesystemChangeObserver)
```

Comment:
viewLifecycleOwner was used for a reason. See [this article](https://medium.com/@BladeCoder/architecture-components-pitfalls-part-1-9300dd969808) for why we use it.